cmd/livepeer_cli/wizard_ticketbroker.go

```diff
@@ -0,0 +1,217 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"math/big"
+	"net/http"
+	"net/url"
+
+	"github.com/golang/glog"
+	"github.com/livepeer/go-livepeer/eth"
+)
+
+func (w *wizard) deposit() {
+	sender, err := w.senderInfo()
+	if err != nil {
+		glog.Errorf("Error getting sender info: %v", err)
+		return
+	}
+
+	params, err := w.ticketBrokerParams()
+	if err != nil {
+		glog.Errorf("Error getting TicketBroker params: %v", err)
+		return
+	}
+
+	reqPenaltyEscrowAmount := big.NewInt(0)
+	if params.MinPenaltyEscrow.Cmp(sender.PenaltyEscrow) > 0 {
+		reqPenaltyEscrowAmount = new(big.Int).Sub(params.MinPenaltyEscrow, sender.PenaltyEscrow)
+	}
+
+	if reqPenaltyEscrowAmount.Cmp(big.NewInt(0)) > 0 {
+		w.fundAndApproveSigners(sender.Deposit, sender.PenaltyEscrow, reqPenaltyEscrowAmount)
+		return
+	}
+
+	w.fundDeposit(sender.Deposit)
+
+	return
+}
+
+func (w *wizard) fundAndApproveSigners(currDeposit *big.Int, currPenaltyEscrow *big.Int, reqPenaltyEscrowAmount *big.Int) {
+	fmt.Printf("Current Deposit: %v\n", currDeposit)
+	fmt.Printf("Current Penalty Escrow: %v\n", currPenaltyEscrow)
+
+	fmt.Printf("Enter deposit amount in ETH (%v will be used cover the minimum penalty escrow) - ", eth.FormatUnits(reqPenaltyEscrowAmount, "ETH"))
+
+	amount := w.readFloatAndValidate(func(in float64) (float64, error) {
+		amountWei := eth.ToBaseUnit(big.NewFloat(in))
+
+		if amountWei.Cmp(reqPenaltyEscrowAmount) < 0 {
+			return 0, fmt.Errorf("must deposit at least %v to cover the minimum penalty escrow", eth.FormatUnits(reqPenaltyEscrowAmount, "ETH"))
+		}
+
+		return in, nil
+	})
+
+	form := url.Values{
+		"amount": {eth.ToBaseUnit(big.NewFloat(amount)).String()},
+	}
+	fmt.Println(httpPostWithParams(fmt.Sprintf("http://%v:%v/fundAndApproveSigners", w.host, w.httpPort), form))
+}
+
+func (w *wizard) fundDeposit(currDeposit *big.Int) {
+	fmt.Printf("Current Deposit: %v\n", currDeposit)
+
+	fmt.Printf("Enter deposit amount in ETH - ")
+
+	amount := w.readFloat()
```

Comment:
I think this means we're allowing users to input a negative number?

Reply:
Good point - I'll make an update to only accept positive numbers

Reply:
Updated to use the [readPositiveFloat()](https://github.com/livepeer/go-livepeer/pull/658/files#diff-5d52c4c551172dd0fedcbb693fdda67eR193) and [readPositiveFloatAndValidate()](https://github.com/livepeer/go-livepeer/pull/658/files#diff-5d52c4c551172dd0fedcbb693fdda67eR203) helpers